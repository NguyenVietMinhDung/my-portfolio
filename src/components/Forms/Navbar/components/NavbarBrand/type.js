export type Props = {
  logoUrl: string,
};
