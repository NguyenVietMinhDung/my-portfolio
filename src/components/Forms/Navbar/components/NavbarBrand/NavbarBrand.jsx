import React from 'react';
import type { NavbarBrandProps } from './type';

const NavbarBrand = (props: NavbarBrandProps) => {
  const { url } = props;
  return <img className="brand pointer" src={url} alt="" />;
};

export default NavbarBrand;
