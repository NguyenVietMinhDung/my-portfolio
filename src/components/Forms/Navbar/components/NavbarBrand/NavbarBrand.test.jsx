import React from 'react';
import { shallow } from 'enzyme';
import logoUrl from '../../../../../assets/images/logo.png';
import NavbarBrand from './NavbarBrand';

const defaultProps = {
  logoUrl,
};

describe('<NavbarBrand />', () => {
  test('renders correctly', () => {
    const component = shallow(<NavbarBrand {...defaultProps} />);
    expect(component).toMatchSnapshot();
  });
});
