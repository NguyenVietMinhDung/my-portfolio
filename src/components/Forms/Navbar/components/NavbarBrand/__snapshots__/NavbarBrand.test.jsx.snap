// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarBrand /> renders correctly 1`] = `
<img
  alt=""
  className="nav-brand"
  src="test-file-stub"
/>
`;
